src/Common/nunit/CommandLineOptions.cs

```diff
@@ -296,6 +298,23 @@ protected virtual void ConfigureOptions()
             this.Add("where=", "Test selection {EXPRESSION} indicating what tests will be run. See description below.",
                 v => WhereClause = RequiredValue(v, "--where"));
 
+            this.Add("params|D=", "Define a test parameter.",
```

Comment:
I am okay with both forms, but I probably lean towards `--define|-d`. To me `--params` made me think that they would be passed in as parameters to test methods or fixtures. If we switch it to `--define`, should we rename the `TestParameters` class and the property on `TestContext` so they all match to prevent confusion? Maybe `TestDefines` and `TestContext.Defines`? Doesn't read as well, but I like consistency.